config/application_routes.js

```diff
@@ -11,6 +11,8 @@ routes.use("/bill", require("./routes/bill_routes"))
 routes.use("/bill_on_demand", require("./routes/bill_on_demand_routes"))
 routes.use("/bill_rate", require("./routes/bill_rate_routes"))
 routes.use("/bill_hub", require("./routes/bill_hub_routes"))
+routes.use("/geographic_site", require("./routes/geographical_site_management_routes"))
+routes.use("/payment_method", require("./routes/Payment_method_routes"))
```

Comment:
The standard is all lower case here. Please update this.

Reply:
done